packages/legend-graph/src/graph/BasicModel.ts

```diff
@@ -614,10 +614,7 @@ export abstract class BasicModel {
    * expose it. The other benefit of exposing this in the main graph is that we could
    * do better duplicated path check
    */
-  protected renameOwnElement(
-    element: PackageableElement,
-    newPath: string,
-  ): void {
+  renameOwnElement(element: PackageableElement, newPath: string): void {
     const elementCurrentPath = element.path;
```

Comment:
can we also add a video on renaming a function. could sense we need some user validation there.